pypit/arpixels.py

```diff
@@ -0,0 +1,223 @@
+""" Routines related to mapping pixels to physical positions
```

Comment:
Not something we should do now (or possibly ever), but I was planning to suggest that we rethink the general pypit naming convention.  I think X has told me its origin, but I think the inclusion of the 'ar' in the front of every file name is not particularly informative.  I think we should be more intentional about what we call the modules and think through a directory structure that will help organize the modules.  When we/someone imports something from pypit, just the name of the module should give you a good indication of what type of function/object you're importing.  Just a suggestion.

Reply:
I agree and am raising it as an issue.